integreat_cms/cms/utils/stringify_list.py

```diff
@@ -3,23 +3,30 @@
 from django.utils.translation import gettext_lazy as _
 
 
-def iter_to_string(iterable: Iterable, *, quotation_char: str = '"') -> str:
+def iter_to_string(
+    iterable: Iterable, *, quotation_char: str = '"', max_items: int = 5
+) -> str:
     """
     This is a helper function that turns a list into a string using different delimiters.
 
     :param iterable: List of items
     :param quotation_char: The character that is added around each item in the list
+    :param max_items: The maximum number of items in the iterable to display
     :return: Returns string with the single items separated by comma expect the last item which is separated by 'and'.
     """
     # Convert iterable to list to support querysets etc.
     lst = list(iterable)
```

Comment:
```suggestion
    list_of_elements = list(iterable)
```

I'd like to make this a bit more specific and follow Robert C. Martin's recommendation in Clean Code chapter 2.5 to use pronounceable names as variables

Reply:
I would also take the subchapter 'avoid encodings' of that book into account, and propose to just rename it to 'elements'. Do you agree?